data/avro-schema/src/main/resources/avro/net/corda/data/messaging/mediator/MediatorReplayOutputEventType.avsc

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "record",
+  "name": "MediatorReplayOutputEventType",
+  "namespace": "net.corda.data.messaging.mediator",
+  "doc": "Record the details of output events to be replayed if necessary.",
+  "fields": [
+    {
+      "name": "value",
+      "type": "bytes",
+      "doc": "Serialized bytes of the object."
+    },
+    {
+      "name": "type",
+      "type": {
+        "type": "enum",
+        "name": "SerializationType",
+        "symbols": ["STRING", "BYTEARRAY", "AVRO"]
```

Comment:
Not directly related to this PR, but what was the reason to handle `String` and `ByteArray` differently than other types when serializing to Avro? (We wouldn't need this class if all serialized objects contain type info)